front/lib/swr.ts

```diff
@@ -554,6 +554,54 @@ export function useDataSourceViewContentNodes({
   };
 }
 
+export function useDataSourceViewContentNodeChildren({
```

Comment:
can we add TODO(GROUPS_INFRA) todos to remove the old hooks that will be superseded by this one in this file :pray:

Reply:
Those TODOs already exist today 😉

Reply:
yay